microk8s-resources/actions/enable.falco.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+microk8s enable dns
+
+NAMESPACE_PTR="sysdig-agent"
+
+MANIFEST_PTR="https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-service.yaml"
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+microk8s enable helm3
+
+# make sure the "sysdigagent" namespace exists
+$KUBECTL create namespace "$NAMESPACE_PTR" > /dev/null 2>&1 || true
+
+# load the CRD and wait for it to be installed
+$KUBECTL apply -f "$MANIFEST_PTR" -n "$NAMESPACE_PTR"
+
+echo "Enabling Auditlogging using webhook"
+mkdir -p "${SNAP_DATA}/args/auditlogging" >/dev/null 2>&1
+cp "${SNAP}/actions/kube-api-audit.yaml" "${SNAP_DATA}/args/auditlogging"
+cp "${SNAP}/actions/webhook-config.yaml" "${SNAP_DATA}/args/auditlogging"
+echo "Reconfiguring apiserver"
+refresh_opt_in_config "audit-log-maxbackup" "3" kube-apiserver
+refresh_opt_in_config "audit-log-maxsize" "1024" kube-apiserver
+refresh_opt_in_config "audit-log-maxage" "30" kube-apiserver
+refresh_opt_in_config "audit-log-path" "/var/log/kube-apiserver-audit.log" kube-apiserver
```

Comment:
Let's not polute the host filesystem. Should we keep the MicroK8s logs under SNAP_COMON.